### PR TITLE
Docs vuepress v1

### PR DESCRIPTION
**Pull request by lauriejim**
#### Description of what you did:
In this PR I updated the docs to Vuepress 1.x and made some changes to match with this new version.

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [x] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: docs/3.0.0-beta.x/concepts/models.md [hunk 1]
```diff
@@ -645,7 +635,7 @@ Callbacks on:
 
 :::
 
-::: tab "destroy" id="destroy"
+::: tab destroy
 
 destroy`
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/concepts/models.md]: missing ` before the word

File: docs/3.0.0-beta.x/getting-started/install-requirements.md [hunk 2]
```diff
@@ -14,11 +14,9 @@ You can install `Node.js` and `npm` for `Windows 10`, `Ubuntu 18.04` and `Mac O/
 
 #### Installation Instructions for each Operating System:
 
-:::: tabs cache-lifetime="10" :options="{ useUrlFragment: false }"
+:::: tabs
 
-::: tab "Windows 10" id="windows-requirements"
-
-**WINDOWS 10**
+::: tab "Windows 10"
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/getting-started/install-requirements.md]: Sometimes we use WIndows other times windows 10. We should be consistent everywhere

File: docs/3.0.0-beta.x/guides/slug.md [hunk 3]
```diff
@@ -80,7 +80,7 @@ module.exports = {
 
 :::
 
-::: tab "Bookshelf" id="bookshelf"
+::: tab Bookshelf
 
 ```js
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/guides/slug.md]: The end of this code block is missing

**Comment by derrickmehaffy** [comment]
I think it's a good idea to remove the v1 docs nice one @lauriejim

**Pull request merged**